Formula/docker-machine-parallels.rb

```diff
@@ -5,6 +5,7 @@ class DockerMachineParallels < Formula
       tag:      "v2.0.1",
       revision: "a1c3d495487413bdd24a562c0edee1af1cfc2f0f"
   license "MIT"
+  revision 1
```

Comment:
Since nothing changes for users I don't think this is needed

Reply:
Tnx, removed it in 58d208cc9fef93ff3ae178ed6a2d90ebd09faf9d, since rebuild and update is not really required.